rfcs/rfc-external-project-and-gem-docs.md

```diff
@@ -0,0 +1,54 @@
+# Establish process for external project and Gem documentation
+
+## Summary
+
+This RFC proposes to clarify whether documentation about external projects and Gems can be included in O3DE Documentation (`o3de.org/docs`). For the context of this RFC, "external" refers to those that live outside of the `o3de` repository.
+
+External projects and Gems commonly fall under the following scenarios:
+- They live outside of the `o3de` organization, such as those created by O3DE partners.
+- Their repositories are part of the `o3de` organization, but not part of the `o3de` repository.
+
+Examples include docs for templates and Gems in `o3de-extras` repo, and tutorials for the Multiplayer Sample Project. 
```

Comment:
```suggestion
Examples include docs for templates and Gems in the `o3de/o3de-extras` repo, and tutorials for the Multiplayer Sample Project (`o3de/o3de-multiplayersample`). 
```

Reply:
I removed `o3de-extras` as an example because this is canonical. Instead I added Atom Sample Viewer as an example.